workflows/request_next_topic.ts

```diff
@@ -0,0 +1,56 @@
+import { DefineWorkflow, Schema } from "deno-slack-sdk/mod.ts";
+import { SendRequestToSpeakerFunction } from "../functions/request_next_topic_definition.ts";
+
+/**
+ * A Workflow composed of two steps: asking for details from the user,
+ * and then forward the details along with two buttons (approve and deny) to the selected speaker.
+ */
+export const Request_next_topic = DefineWorkflow({
```

Comment:
we need to add this to manifest so we can run it on slack
also we should follow existing naming format and rename this to something like `RequestNextTopic`